proto/ibc/core/packetserver/v1/counterparty.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package ibc.core.packetserver.v1;
+
+option go_package = "github.com/cosmos/ibc-go/v9/modules/core/packet-server/types";
+
+import "gogoproto/gogo.proto";
+import "ibc/core/commitment/v2/commitment.proto";
+
+// Counterparty defines the counterparty for a light client to implement IBC eureka protocol
+// this value is stored under our side's channel ID. which we will use to write all packet messages
+// sent to counterparty
+message Counterparty {
```

Comment:
Think it'd be nice to see this `Counterparty` type as something used by historical situations (channel/port), but not necessarily only for that